diff --git a/node_modules/ecctrl/dist/Ecctrl.d.ts b/node_modules/ecctrl/dist/Ecctrl.d.ts
index 69a3dac..faa55cd 100644
--- a/node_modules/ecctrl/dist/Ecctrl.d.ts
+++ b/node_modules/ecctrl/dist/Ecctrl.d.ts
@@ -4,7 +4,7 @@ import React from "react";
 export { EcctrlAnimation } from "./EcctrlAnimation";
 export { useFollowCam } from "./hooks/useFollowCam";
 export { useGame } from "./stores/useGame";
-export { EcctrlJoystick } from "../src/EcctrlJoystick";
+export { EcctrlJoystick } from "./EcctrlJoystick";
 export { useJoystickControls } from "./stores/useJoystickControls";
 declare const _default: React.ForwardRefExoticComponent<EcctrlProps & React.RefAttributes<CustomEcctrlRigidBody>>;
 export default _default;
diff --git a/node_modules/ecctrl/src/Ecctrl.tsx b/node_modules/ecctrl/src/Ecctrl.tsx
index 052552c..a651121 100644
--- a/node_modules/ecctrl/src/Ecctrl.tsx
+++ b/node_modules/ecctrl/src/Ecctrl.tsx
@@ -26,7 +26,7 @@ import React from "react";
 export { EcctrlAnimation } from "./EcctrlAnimation";
 export { useFollowCam } from "./hooks/useFollowCam";
 export { useGame } from "./stores/useGame";
-export { EcctrlJoystick } from "../src/EcctrlJoystick";
+export { EcctrlJoystick } from "./EcctrlJoystick";
 export { useJoystickControls } from "./stores/useJoystickControls";
 
 // Retrieve current moving direction of the character
